test/Generators/Microsoft.Gen.Logging/TestClasses/NonStaticTestClasses.cs

```diff
@@ -43,7 +43,33 @@ public partial class LoggerInNullablePropertyDerivedTestClass : LoggerInNullable
         public partial void M1(string p0);
     }
 
-    public partial class GenericLoggerInPropertyDerivedTestClass : LoggerInNullablePropertyTestClass
+    // The following classes are not used in any tests, but they validate that the generation is working correctly.
```

Comment:
Why not? The point of these generated tests is that you can actually test that the code gen worked as you expected.